Mathlib/Topology/MetricSpace/HolderNorm.lean

```diff
@@ -225,7 +225,8 @@ lemma eHolderNorm_add_le :
     obtain (h | h) := hfg
     all_goals simp [h]
 
-lemma eHolderNorm_smul {α} [NormedDivisionRing α] [Module α Y] [IsBoundedSMul α Y] (c : α) :
+-- TODO: does this work with `NormedRing`?
```

Comment:
I think the answer is yes, after #24464 and an `Iff` lemma using the new typeclass.